tinnsleep/scoring.py

```diff
@@ -122,40 +139,101 @@ def get_event_label(episode):
         return 3
 
 
-def create_list_events(li_ep, interval):
+def create_list_events(li_ep, time_interval, recording_duration):
     """ Creates the list of events, 0 = no event, 1 = tonic episode, 2 = phasic 
     episode, 3 = mixed episode
     
     Parameters
     ----------
     li_ep : list of episode instances
-    interval: float, interval between 2 elementary events
+    time_interval: float, time interval in seconds between 2 elementary events
+    recording_duration : float, duration of the recording in seconds
     Returns
     -------
     list of integers, labels of the events 
     """
-    #Deals the case the input is empty
+    # Deals the case the input is empty
     li_events = []
     if len(li_ep) == 0:
         return []
 
     # Initialization
     # Case where the first burst does not begin at time=0
-    if li_ep[0].beg / interval > 1:
-        li_0 = [0 for i in range(int(li_ep[0].beg / interval))]
+    if li_ep[0].beg / time_interval > 1:
+        li_0 = [0 for i in range(int(li_ep[0].beg / time_interval))]
         li_events.extend(li_0)
 
     # Tagging the first event
     label = get_event_label(li_ep[0])
-    li_events.extend([label for i in range(int((li_ep[0].end - li_ep[0].beg) / interval))])
+    li_events.extend([label for i in range(int((li_ep[0].end - li_ep[0].beg) / time_interval))])
 
     # Loop of tagging
     if len(li_ep) > 1:
         for i in range(len(li_ep) - 1):
             # Putting zeros between two episodes
-            li_events.extend([0 for i in range(int((li_ep[i + 1].beg - li_ep[i].end) / interval))])
+            li_events.extend([0 for i in range(int((li_ep[i + 1].beg - li_ep[i].end) / time_interval))])
             # Tagging the next episode
             label = get_event_label(li_ep[i + 1])
-            li_events.extend([label for i in range(int((li_ep[i + 1].end - li_ep[i + 1].beg) / interval))])
+            li_events.extend([label for i in range(int((li_ep[i + 1].end - li_ep[i + 1].beg) / time_interval))])
 
+    # If necessary, adds 0s at the end of li_events until the end of the recording
+    if (recording_duration - li_ep[-1].end) / time_interval > 1.0:
+        li_0 = [0 for i in range(int((recording_duration - li_ep[-1].end) / time_interval))]
+        li_events.extend(li_0)
     return li_events
```

Comment:
You did great to refactor `interval` and `duration` however I advice consistency. 

why use `time_` then `recording_` as parameters keyword?

Choose one keyword and change parameter name if the function is different, e.g.
`time_duration`: always an epoch length
`time_interval`: always the length between two starts of epoch
`time_recording`: always the total length of a recording